src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/config.jelly

```diff
@@ -8,9 +8,9 @@
       </f:entry>
     </j:when>
     <j:otherwise>
-      <f:invisibleEntry>
-        <f:select field="apiUri"/>
-      </f:invisibleEntry>
+      <f:entry title="${%API endpoint}" field="apiUri">
+        <f:textbox readonly="readonly"/>
+      </f:entry>
```

Comment:
This change is really weird. On a pristine instance, I get this uneditable text field in an org folder configuration:

<img width="2197" height="554" alt="Screenshot 2025-08-29 at 17 12 25" src="https://github.com/user-attachments/assets/ee3536c0-6370-44f3-90d6-98e738ef8804" />

Why this change?